### PR TITLE
[CI] use urbit_amd64.deb from urbit/urbit release

### PR DESCRIPTION
**Pull request by ohAitch**

### CODE REVIEW

File: .travis/pin-urbit-release.url [hunk 1]
```diff
@@ -1 +0,0 @@
-https://github.com/urbit/urbit/releases/download/redefault-0.5.1
```
Comment (belisarius222) [inline .travis/pin-urbit-release.url]: I liked having a pinned release URL in a single file. Why move it to inline?

**Comment by ohAitch** [comment]
Fair, I think I was aiming for something like "make it more of a hassle to
push an arvo that doesn't work on the deployed network" which is never a
good motivation

(Or like, makes sense on master, but when we inevitably get a next-breach
branch it should keep the old convention)

**Comment by belisarius222** [comment]
Ping. Can you please reinstate the `pin-urbit-release.url` file? Then I'll merge this.

**Comment by ohAitch** [comment]
Right, thanks

**Comment by belisarius222** [comment]
@ohAitch any idea why the build failed? It looks like it couldn't apt install the .deb file for some reason.

**Comment by ohAitch** [comment]
yup, I mixed up wget -o and -O, seems to be going fine now

**Pull request merged**